Makefile

```diff
@@ -4,7 +4,7 @@ export DOCKER_TAGNAME ?= latest
 .PHONY: license
 license: $(TOOLBIN)/license_finder
 	$(call license_go,.)
-	$(call license_python,secret-provider)
+	$(call license_python)
```

Comment:
we don't have any python code after this PR right?

Reply:
I do not see any  .py files... should I remove license_python code?

Reply:
yes